faiss/gpu/utils/CuvsUtils.h

```diff
@@ -26,32 +26,32 @@
 #include <faiss/gpu/GpuResources.h>
 #include <faiss/gpu/utils/Tensor.cuh>
 
-#include <raft/distance/distance_types.hpp>
+#include <cuvs/distance/distance.h>
 
 #pragma GCC visibility push(default)
 namespace faiss {
 namespace gpu {
 
-inline raft::distance::DistanceType metricFaissToRaft(
+inline cuvsDistanceType metricFaissToCuvs(
         MetricType metric,
         bool exactDistance) {
     switch (metric) {
         case MetricType::METRIC_INNER_PRODUCT:
-            return raft::distance::DistanceType::InnerProduct;
+            return cuvsDistanceType::InnerProduct;
         case MetricType::METRIC_L2:
-            return raft::distance::DistanceType::L2Expanded;
+            return cuvsDistanceType::L2Expanded;
         case MetricType::METRIC_L1:
-            return raft::distance::DistanceType::L1;
+            return cuvsDistanceType::L1;
         case MetricType::METRIC_Linf:
-            return raft::distance::DistanceType::Linf;
+            return cuvsDistanceType::Linf;
         case MetricType::METRIC_Lp:
-            return raft::distance::DistanceType::LpUnexpanded;
+            return cuvsDistanceType::LpUnexpanded;
         case MetricType::METRIC_Canberra:
-            return raft::distance::DistanceType::Canberra;
+            return cuvsDistanceType::Canberra;
         case MetricType::METRIC_BrayCurtis:
-            return raft::distance::DistanceType::BrayCurtis;
+            return cuvsDistanceType::BrayCurtis;
         case MetricType::METRIC_JensenShannon:
-            return raft::distance::DistanceType::JensenShannon;
+            return cuvsDistanceType::JensenShannon;
         default:
             RAFT_FAIL("Distance type not supported");
```

Comment:
Should this be a CUVS error message?

Reply:
We currently use `RAFT_FAIL` as an error generator in cuVS too. We currently don't have a cuVS error message but I have marked it as a TODO. This certainly needs to be done to avoid confusion since it is a cuVS failure and not a RAFT failure.